test/utils.test.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-var Buffer = require('safe-buffer').Buffer;
-var utils = require('../lib/utils');
-var assert = require('assert');
-var debug = require('debug');
+const Buffer = require('safe-buffer').Buffer;
+const utils = require('../lib/utils');
+const assert = require('assert');
```

Comment:
if you want to target node 12 then this if fine but if you set the lowest support to 12.20 then you can use `node:assert` 

kind of nice to know when something is a node-builtin package/module

Reply:
I suppose that would be nice, thanks for the info. But I don't think it's worth narrowing our version range for a change that's just for internal readability.

Reply:
That is fine, it's just something i have notices that many who switches to esm (12.17) also go +3 minor version extra (12.20) just to get that extra `node:` prefix, most of it comes from the xo linter that now recommends everyone to switch to esm when using the latest version of xo

Reply:
can resolve this if u want to keep using >=12.0.0